.travis.yml

```diff
@@ -9,6 +9,7 @@ python:
 install:
   #- pip install .
   #- pip install -r requirements.txt
+  - pip install --upgrade pip
```

Comment:
we should probably pip freeze and restore the `-r` command above sometime and disable the `install` ones below this.